stable_baselines3/common/sb2_compat/rmsprop_tf_like.py

```diff
@@ -0,0 +1,126 @@
+import torch
+from torch.optim import Optimizer
+
+
+class RMSpropTFLike(Optimizer):
```

Comment:
I think we are missing a test for that one

Reply:
and documentation

Reply:
What'd you like to see for this one? The documentation for the optimizer is in the docstring, but do you mean the recommendation in the docs. As for the tests is it enough to have functionality check (i.e. it does not crash / it updates parameters), or something more in-depth? 

I will add notes about this into #123, too.

Reply:
yes, recommendation in the A2C doc and in the migration guide sounds good ;)

yes, at least that it runs.